web/src/test/java/org/cbioportal/weblegacy/GenePanelControllerTest.java

```diff
@@ -121,16 +121,30 @@ public void genePanelByStableIdTest() throws Exception {
     }
 
     @Test
```

Comment:
(optional)  For list formatting testing, I would recommend adding another sample and another gene to the mocked service return (2 samples, 2 genes). Or maybe add a second test which returns results with more than one member in the list.